framework/filters/AccessRule.php

```diff
@@ -90,6 +92,62 @@ class AccessRule extends Component
      */
     public $denyCallback;
 
+    /** 
+     * @var array a map for passing parameters to the [[User::can()]] function
+     * 
+     * The map indicates query parameters that will be passed when determining
+     * role access.
+     * 
+     * If both a key and a value are passed, then the name of the parameter will
+     * be changed. A value of `['a' => 'b']` will cause [[User::can()]] to be
+     * invoked with the content of the request query parameter 'a' as key 'b'
+     * in the params argument.
+     * If only a value is passed then it is equivalent to passing the same value
+     * as key.
+     * 
+     * The special parameters '{action}' and '{controller}' are bound to the
```

Comment:
i think `{route}` would be better since i can have `api/v1/user/index` and `api/v2/user/index` with different permissions.